CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+Hi! 
+
+Thanks for considering participation in WarsawLO project!
+We believe that tools like ours should be crafted by the community! 
```

Comment:
We, the maintainers, believe that tools like this one should be crafted by a community.